packages/app/src/core/plugin-assembler.ts

```diff
@@ -15,17 +15,17 @@
  */
 
 import * as Debug from 'debug'
-const debug = Debug('core/plugin-assembler')
-debug('loading')
 
 import * as fs from 'fs-extra'
 import * as path from 'path'
 import * as events from 'events'
-import mkdirp = require('mkdirp')
 import { exec } from 'child_process'
 
 import * as plugins from './plugins'
 import * as commandTree from './command-tree'
+const debug = Debug('core/plugin-assembler')
+debug('loading')
+import mkdirp = require('mkdirp')
```

Comment:
this one all seems a little nonsensical to me. why is the import of mkdirp treated not as an import?

Reply:
Hum, maybe it is related to the fact that this variable is not being used?
@jesusprubio is working on that rule.

<img width="549" alt="Screenshot 2019-06-04 at 14 33 01" src="https://user-images.githubusercontent.com/1007051/58879312-aff90780-86d5-11e9-9201-05483ba7e1c9.png">

Here `events`, `exec`, `mkdirp`, `TMP` and `TMA` are not being used.

Reply:
oh great!